### ISSUE TITLE
Clicking on application title leads to non-existing URL

### ISSUE THREAD

**Issue by dietervu** [clarin-eric/fcs-sru-aggregator#11]
When accessing the aggregator via https://contentsearch.clarin.eu , clicking on the application title leads to https://contentsearch.clarin.eu/ws/fcs/2.0/aggregator/ which results in a blank page.

**Comment by andmor-** [comment]
I can work around it with a redirect on the central proxy:
https://gitlab.com/CLARIN-ERIC/compose_clarin_proxy/commit/a004d5c5ce3c5043707c94da750366ec06d99115

Though this is only needed because the FCS sets the link on the application title in an absolute fashion, and based on the request's "Host" header (which we set to `spraakbanken.gu.se`).

The proposed fix is deployed.

**Comment by dietervu** [comment]
Unless @ljo can fix this quickly (before 29 August) I suggest to make use of andmor's workaround.

We should in any case fix this at the FCS level too, so maybe best to keep this issue open.

**Comment by dietervu** [comment]
Giving this issue a bump since I believe something similar is still going on.
For the record, the gitlab fragment mentioned above (since it requires an account there):

``` listen       443;
        server_name contentsearch.clarin.eu;

        location /ws/fcs/2.0/aggregator/ {
            return 301 https://$host;
        }

        location / {
            proxy_ssl_server_name on;   #Enable SNI
            proxy_set_header        Host "spraakbanken.gu.se";
```
When I start browsing the current [production version](https://contentsearch.clarin.eu) (v.3.0.2-67), the leftmost "home" link to Federated Content Search leads to:

```
<a class="navbar-brand" href="/Aggregator/" tabindex="-1"><img src="img/magglass1.png" width="28px" height="28px"><header class="inline"> Federated Content Search </header></a>
```
I believe this /Aggregator path is a remainder of the past and should be removed. After clicking multiple times in the interface, in the end it leads to paths like /Aggregator/Aggregator, which then results in the following error message:

```
HTTP ERROR 400 Duplicate valid session cookies: node0bglek9utij014if5h9aa87bk34.node0,node0p1redvsthw041rwgviw7m1b2f35.node0
URI:	/Aggregator/Aggregator/
STATUS:	400
MESSAGE:	Duplicate valid session cookies: node0bglek9utij014if5h9aa87bk34.node0,node0p1redvsthw041rwgviw7m1b2f35.node0
SERVLET:	-
CAUSED BY:	org.eclipse.jetty.http.BadMessageException: 400: Duplicate valid session cookies: node0bglek9utij014if5h9aa87bk34.node0,node0p1redvsthw041rwgviw7m1b2f35.node0
Caused by:

org.eclipse.jetty.http.BadMessageException: 400: Duplicate valid session cookies: node0bglek9utij014if5h9aa87bk34.node0,node0p1redvsthw041rwgviw7m1b2f35.node0
	at org.eclipse.jetty.server.session.SessionHandler.checkRequestedSessionId(SessionHandler.java:1675)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1526)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1212)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at com.codahale.metrics.jetty9.InstrumentedHandler.handle(InstrumentedHandler.java:249)
	at io.dropwizard.jetty.ContextRoutingHandler.handle(ContextRoutingHandler.java:37)
	at org.eclipse.jetty.server.handler.gzip.GzipHandler.handle(GzipHandler.java:767)
	at org.eclipse.jetty.server.handler.RequestLogHandler.handle(RequestLogHandler.java:54)
	at org.eclipse.jetty.server.handler.StatisticsHandler.handle(StatisticsHandler.java:173)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
	at org.eclipse.jetty.server.Server.handle(Server.java:500)
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:547)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:270)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129)
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:388)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.lang.Thread.run(Thread.java:745)
```
I can replicate this in a private browser window, so I do not believe it is related to "old session cookies" hanging around on my computer.

**Comment by Querela** [comment]
Yes, but I still wanted to wait to close this until I have merged #51 into `master`.
